src/mqueryfront/src/QueryField.js

```diff
@@ -12,13 +12,22 @@ class QueryField extends Component {
         });
 
         var multiselect = null;
+        var placeHolder = "everywhere";
+
+        if (this.props.selectedTaints.length) {
```

Comment:
is this check necessary? if the state is always a list (empty or non-empty) it shouldn't be necessary.